shared/src/test/resources/vulndep.json

```diff
@@ -0,0 +1,399 @@
+                    "sha1": "B7C3AB36056B98CBE0AA098597B511B80661DABF",
+                    "digestAlgorithm": "SHA1",
+                    "createdAt": "2019-08-19T21:41:20.420+0000",
+                    "modifiedAt": "2019-08-19T21:41:20.420+0000",
+                    "libraryId": {
+                        "artifact": "vulas-testapp-coreapi",
+                        "version": "3.1.5-MVN",
+                        "group": "com.acme.foo"
+                    },
+                    "bundledLibraryIds": [
+                        {
+                            "artifact": "vulas-testapp-coreapi",
+                            "version": "3.1.5-MVN",
+                            "group": "com.acme.foo"
+                        }
+                    ],
+                    "wellknownDigest": false,
+                    "digestTimestamp": null,
+                    "digestVerificationUrl": null
+                },
+                "parent": null,
+                "origin": null,
+                "declared": true,
+                "traced": false,
+                "scope": "COMPILE",
+                "transitive": false,
+                "filename": "vulas-testapp-coreapi-3.1.5-MVN.jar",
+                "path": "\\.m2\\repository\\com\\acme\\foo\\vulas-testapp-coreapi\\3.1.5-MVN\\vulas-testapp-coreapi-3.1.5-MVN.jar",
+                "relativePath": null
+            },
+            "origin": null,
+            "declared": true,
+            "traced": false,
+            "scope": "COMPILE",
+            "transitive": true,
+            "filename": "vulas-testapp-core-3.1.5-MVN.jar",
+            "path": "\\.m2\\repository\\com\\acme\\foo\\vulas-testapp-core\\3.1.5-MVN\\vulas-testapp-core-3.1.5-MVN.jar",
+            "relativePath": null
+        },
+        "origin": null,
+        "declared": true,
+        "traced": false,
+        "scope": "SYSTEM",
+        "transitive": true,
+        "filename": "cf1.2.2-cc1.4.jar",
+        "path": "\\Documents\\projects\\vulas\\testapp-java\\core\\lib\\cf1.2.2-cc1.4.jar",
+        "relativePath": null
+    },
+    "bug": {
+        "bugId": "CVE-2014-0050",
+        "bugIdAlt": null,
+        "maturity": "READY",
+        "origin": "PUBLIC",
+        "description": "MultipartStream.java in Apache Commons FileUpload before 1.3.1, as used in Apache Tomcat, JBoss Web, and other products, allows remote attackers to cause a denial of service (infinite loop and CPU consumption) via a crafted Content-Type header that bypasses a loop's intended exit conditions.",
+        "descriptionAlt": null,
+        "cvssScore": 7.5,
+        "cvssVersion": "2.0",
+        "cvssVector": "AV:N/AC:L/Au:N/C:P/I:P/A:P",
+        "reference": [],
+        "createdAt": "2016-12-16T15:42:12.251+0000",
+        "createdBy": null,
+        "modifiedAt": "2019-03-14T23:29:23.466+0000",
+        "modifiedBy": null,
+        "cvssDisplayString": "7.5 (v2.0)"
+    },
+    "constructList": [
```

Comment:
I am not sure about how this test file was created but I think it comes from running the code as in the serialization of the vulnerableDependency the View to be used was missing and thus the constructList was serialized even if we never did it as it's not required by the client at this step.